server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsIndices.java

```diff
@@ -185,11 +206,11 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
      * @opensearch.api
      */
     @PublicApi(since = "1.0.0")
-    public static class ShardStats implements ToXContentFragment {
+    public static class ShardStats implements ToXContentFragment, Writeable {
```

Comment:
Making this Writeable as we create this in `AggregatedNodeLevelStats` on each node and return it from there. The coordinating node will then just merge the results from each node instead of going over each `indices.stats.ShardStats`.

As this now comes in the ser/de path, needs to made Writeable.